api-docs/api-docs.yml

```diff
@@ -0,0 +1,540 @@
+          type: object
+          additionalProperties:
+              x-additionalPropertiesName: <ENTITY_TYPE> or DEFAULT
+              type: object
+              anyOf:
+                - $ref: "#/components/schemas/Replace"
+                - $ref: "#/components/schemas/Redact"
+                - $ref: "#/components/schemas/Mask"
+                - $ref: "#/components/schemas/Hash"
+          default:
+            {"DEFAULT": { "type": "replace", "new_value": "<ENTITY_TYPE>" }}
+
+        analyzer_results:
+          type: array
+          description: "Array of analyzer detections"
+          items:
+            $ref: "#/components/schemas/AnalyzerResult"
+
+    AnalyzerResult:
+      type: object
+      required:
+        - start
+        - end
+        - score
+        - entity_type
+      properties:
+        start:
+          type: integer
+          description: "Where the PII starts"
+          example: 24
+        end:
+          type: integer
+          description: "Where the PII ends"
+          example: 32
+        score:
+          type: number
+          format: double
+          description: "The PII detection score"
+          example: 0.8
+        entity_type:
+          $ref: "#/components/schemas/EntityTypes"
+
+    AnalyzerResultWithAnaysisExplanation:
+      allOf:
+        - $ref: '#/components/schemas/AnalyzerResult'
+        - type: object
+          properties:
+            analysis_explanation:
+              $ref: "#/components/schemas/AnalysisExplanation"
+
+    AnalysisExplanation:
+      type: object
+      properties:
+        recognizer:
+          type: string
+          description: "Name of recognizer that made the decision"
+        pattern_name:
+          type: string
+          description: "name of pattern (if decision was made by a PatternRecognizer)"
+        pattern:
+          type: string
+          description: "Regex pattern that was applied (if PatternRecognizer)"
+        original_score:
+          type: number
+          format: double
+          description: "Recognizer's confidence in result"
+        score:
+          type: number
+          format: double
+          description: "The PII detection score"
+        textual_explanation:
+          type: string
+          description: "Free text for describing a decision of a logic or model"
+        score_context_improvement:
+          type: number
+          format: double
+          # TODO: description: "TODO"
+        supportive_context_word:
+          type: string
+          description: "The context word which helped increase the score"
+        validation_result:
+          type: number
+          format: double
+          description: "Result of a validation (e.g. checksum)"
+
+
+    EntityTypes:
+      description: "The PII detection type"
+      type: string
+      enum:
+        - NAME
+        - FIRST_NAME
+        - LAST_NAME
```

Comment:
Those weren't in supported_entities.md, anything else I'm missing?

Reply:
we support PRESON entity out of the box.